README.event-manager.md

```diff
@@ -103,7 +103,9 @@ may also be an empty string.
 The command is an uzbl command which may use format string replacement:
 
 * `%s`: The string as given.
-* `%r`: The string, escaped and quoted for uzbl.
+* `%r`: The string, escaped for uzbl.
+  This format used to also wrap the string in single-quotes. The same result
+  can still be achived by include the quotes in the format string (e.g `'%r'`)
```

Comment:
There are lots of `%r` instances in the config that probably need updated (primarily the uses in search bindings).